### PR TITLE
Add cmd HighlightColors" with param: on, off & toggle; Del vimscript and Integrate its function into lua

### PR DESCRIPTION
**Pull request by sin1111yi**
Add an user command which can accpet 1 arg to easily config color highlight.

### CODE REVIEW

**Comment by brenoprata10** [comment]
Hey o/
Thanks a lot for your contribution!
Could you explain what is the difference of your code to what is coded here? 
https://github.com/brenoprata10/nvim-highlight-colors/blob/c6fe75e3b3193818d94042de1a21f3e62252e994/plugin/nvim-highlight-colors.vim#L12

**Comment by sin1111yi** [comment]
> Hey o/ Thanks a lot for your contribution! Could you explain what is the difference of your code to what is coded here?
> 
> https://github.com/brenoprata10/nvim-highlight-colors/blob/c6fe75e3b3193818d94042de1a21f3e62252e994/plugin/nvim-highlight-colors.vim#L12

My bad, seems like there are no obvious difference between mine and this code. But mine PR is written in lua and using args to run on, off and toggle. Just close my PR if u prefer the vimscript one. However, if u prefer a lua one, I think maybe this vimscript can be deleted.

**Comment by sin1111yi** [comment]
Update readme, and delete that vimscript. I found only those commands are still in use for now, so I delete the vimscripts. Now u can choice mine as pure lua version, or just close my PR to keep it as it is.

**Comment by brenoprata10** [comment]
Could you also delete the commands here?
https://github.com/brenoprata10/nvim-highlight-colors/blob/c6fe75e3b3193818d94042de1a21f3e62252e994/plugin/nvim-highlight-colors.vim#L13
Let's use the great Lua solution you came up with :)

**Comment by brenoprata10** [comment]
Ah never mind, I see you already deleted the file )

File: lua/nvim-highlight-colors/utils.lua [hunk 1]
```diff
@@ -87,4 +91,14 @@ function M.close_windows (windows)
 	end
 end
 
+function M.deprecate()
+	for _, cmd in ipairs(deprecated.cmd) do
+		vim.api.nvim_create_user_command(cmd,
+			function()
+				print("The command \'" .. cmd .. "\' has been deprecated!")
```
Comment (brenoprata10) [inline lua/nvim-highlight-colors/utils.lua:98]:
```suggestion
				print("The command \'" .. cmd .. "\' has been deprecated! Please use the new syntax: HighlightColors On/Off/Toggle")
```

**Comment by sin1111yi** [comment]
Hi, I tried to del the vimscript and Integrate its function into a lua file, and it works. What's more, I add a message for someone who still using the old command by adding a function in utils.lua. I think now their are no repeat functions. This is my first time to open a PR, thank you for your patience.

**Comment by sin1111yi** [comment]
Thank you for your recognition! :)

**Pull request merged**